mobile/lib/shared/providers/asset.provider.dart

```diff
@@ -14,18 +18,72 @@ import 'package:logging/logging.dart';
 import 'package:openapi/api.dart';
 import 'package:photo_manager/photo_manager.dart';
 
-class AssetNotifier extends StateNotifier<List<Asset>> {
```

Comment:
If you remove `StateNotifier` here any change to the state of the class won't update the state.

Reply:
Have a look at line 66. This is just the "POJO" state class. There still is a notifier.